dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/external/conf/TestDhisConfigurationProvider.java

```diff
@@ -0,0 +1,124 @@
+
+package org.hisp.dhis.external.conf;
+
```

Comment:
Needs source code license and @author info.

Reply:
done